### DISCUSSION TITLE
When is the next Swagger-UI release going to be available

### DISCUSSION THREAD

**Discussion by abutch3r** [swagger-api/swagger-ui#10181] [General]
The Latest available swagger-ui release v5.17.14 was released 5 months ago. Since then there have been a number of automated updates made to Swagger-UI, in particular to dependencies where at least one is required to resolve a CVE in a sub-dependency.

I would like to know when the next release is going to be.

reviewing the versions via `JSON.stringify(versions)` I see the current version of https://editor.swagger.io is `5.17.14` while https://editor-next.swagger.io/ is using `5.21.0`.

**Comment by javac9** [comment]
Version 5.18.0 got released today
